api/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
The api container fails to start because wait-for-it.sh's file permissions don't include execution. You'll need to push a file permission change on the file.

Reply:
changed wait-for-it.sh' to wait-for. Tested on my local

Reply:
I still get a permission error here: `sh: ./wait-for: Permission denied`. On my computer, the wait-for script isn't executable. This should show `-rwxr-xr-x`:
```sh
$ ls -al api/wait-for 
-rw-r--r--  1 jander  staff  4059 May  3 18:59 api/wait-for
```
I'm guessing you're using Windows? I know on a Mac or Linux machine I'd fix this with `chmod +x api/wait-for`, but I'm not familiar enough with Windows file permissions to suggest a solution off the top of my head.

Reply:
Yes, I am using Windows.
I will update the permissions

Reply:
added permissions for the file